test/algorithms/intersection/issue_1439.cpp

```diff
@@ -0,0 +1,39 @@
+#include <boost/geometry.hpp>
+#include <boost/geometry/geometries/point_xy.hpp>
+#include <boost/geometry/geometries/polygon.hpp>
+
+#include <vector>
+
+namespace bg = boost::geometry;
+
+using point = bg::model::d2::point_xy<double>;
+using polygon = bg::model::polygon<point>;
+
+int main()
+{
+    polygon p1, p2;
+
+    bg::read_wkt(
+        "POLYGON((-2.47089026 -86.03059246,"
+        "-1.161944873 146.3030596,"
+        "3.40221214 145.8628015,"
+        "3.024695769 141.949088,"
+        "2.420586453 111.9554564,"
+        "1.208013978 -86.04069936,"
+        "-2.47089026 -86.03059246))", p1);
+
+    bg::read_wkt(
+        "POLYGON((-6.213278056 -88.01851748,"
+        "-6.207382255 -86.51852906,"
+        "-6.20539951 -86.01408604,"
+        "-2.470873099 -86.02434575,"
+        "-2.472221358 -86.53315586,"
+        "-2.476196085 -88.03315059,"
+        "-6.213278056 -88.01851748))", p2);
+
+    std::vector<polygon> output;
+    bg::intersection(p1, p2, output);
+
+    // Expected: no intersection
+    return output.empty() ? 0 : 1;
+}
```

Comment:
Thanks a lot. But yes, please adapt it to our current framework.

The cases themselves can go to:
https://github.com/boostorg/geometry/blob/develop/test/algorithms/overlay/overlay_cases.hpp#L1273

The test can go to:
https://github.com/boostorg/geometry/blob/develop/test/algorithms/set_operations/intersection/intersection.cpp#L297

These tests test on area, so the expected area should be `0.0` 

The test will then fail. So you can put the test between
`#ifdef BOOST_GEOMETRY_TEST_ENABLE_FAILING`
and
`#endif`